### PR TITLE
Task817 striped bars

### PR DESCRIPTION
**Pull request by dancingolav**
task 817:  tests ProgressBarsListTests.java, assertions ProgressAlert, changes in general.xml

### CODE REVIEW

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listprogressbars/ProgressBarsListTests.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    }
+
+    @Test(dataProvider = "progressData")
+    public void checkProgressData(String progressId, String value, String color,
+                                  String min, String max, String classStriped) {
+
+        progressSections.stream().filter(progressSection ->
+                progressSection.progress.attr("id").equals(progressId)).forEach(
+                progressSection -> {
+                    assertThat(progressSection.progress.core().attr("class"), containsString(classStriped));
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listprogressbars/ProgressBarsListTests.java]: can we use something simple like this here: `progressSection.progress.core().hasClass(classStriped);`  ?

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listprogressbars/ProgressBarsListTests.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                                                 .maxValue(max);
+                });
+    }
+
+    @Test
+    public void baseValidationTest() {
+        progressSections.forEach(
+                progressSection ->
+                        baseValidation(progressSection.progress));
+
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listprogressbars/ProgressBarsListTests.java]: empty string

File: jdi-light-bootstrap-tests/src/test/resources/general.xml [hunk 3]
```diff
@@ -3,11 +3,15 @@
...
         <packages>
-            <package name="io.github.epam.html.tests.elements.common"/>
+            <!--<package name="io.github.epam.html.tests.elements.common"/>
             <package name="io.github.epam.html.tests.elements.complex"/>
             <package name="io.github.epam.html.tests.elements.composite.*"/>
             <package name="io.github.epam.html.tests.site"/>
-            <package name="io.github.epam.html.tests.suspend"/>
+            <package name="io.github.epam.html.tests.suspend"/>-->
+
+            <package name="io.github.epam.bootstrap.tests.common"/>
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/resources/general.xml]: may be  `<package name="io.github.epam.bootstrap.tests.*"/>` is more suitable for us here, isn't it?

File: jdi-light-bootstrap-tests/src/test/resources/general.xml [hunk 4]
```diff
@@ -3,11 +3,15 @@
 <suite name="Test suite" parallel="methods" thread-count="1">
     <test name="Tests">
         <packages>
-            <package name="io.github.epam.html.tests.elements.common"/>
+            <!--<package name="io.github.epam.html.tests.elements.common"/>
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/resources/general.xml]: should we remove it now?

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/Progress.java [hunk 5]
```diff
@@ -29,4 +34,4 @@ public String getMinValue() {
     public ProgressAssert is() {
         return new ProgressAssert().set(this);
     }
-}
+}
```
Comment (mf-code) [inline jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/Progress.java]: no new line here

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listprogressbars/ProgressBarsListTests.java (hunk 1 above)
Comment (dancingolav) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listprogressbars/ProgressBarsListTests.java]: ok

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listprogressbars/ProgressBarsListTests.java (hunk 2 above)
Comment (dancingolav) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listprogressbars/ProgressBarsListTests.java]: ok

File: jdi-light-bootstrap-tests/src/test/resources/general.xml (hunk 3 above)
Comment (dancingolav) [reply jdi-light-bootstrap-tests/src/test/resources/general.xml]: ok

File: jdi-light-bootstrap-tests/src/test/resources/general.xml (hunk 4 above)
Comment (dancingolav) [reply jdi-light-bootstrap-tests/src/test/resources/general.xml]: ok

File: jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/Progress.java (hunk 5 above)
Comment (dancingolav) [reply jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/Progress.java]: ok

**Pull request merged**